python/sglang/srt/server.py

```diff
@@ -89,10 +89,34 @@
 tokenizer_manager = None
 
 
+@app.get("/v1/health")
+async def health(request: Request) -> Response:
+    """
+    Generate 1 token to verify the health of the inference service.
+    """
+    gri = GenerateReqInput(text="s", sampling_params={"max_new_tokens": 1, "temperature": 0.7})
+    try:
+        _ = await tokenizer_manager.generate_request(gri, request).__anext__()
+        return Response(status_code=200)
+    except Exception as e:
+        logger.exception(e)
+        return Response(status_code=503)
+
+
 @app.get("/health")
```

Comment:
Don't change `/health`.

Reply:
OMG, that's a mistake, I'm sorry.

Reply:
I'm not very familiar with the `rebase` command. Sorry about that again.